compile_docker.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+
+BASE_IMAGE="ubuntu:jammy-20220421"
+
+IMAGE_NAME="scons_image"
+CONTAINER_NAME="scons_container"
+
+COMPILER_DIR=embedded/platforms/toolchain-gccarmnoneeabi/
+
+# wrapper variables 
+COMPONENT=""
+
+# pull base image 
+docker pull ${BASE_IMAGE}
+	
+# build base ubuntu image  
+docker build -t ${IMAGE_NAME} .
+
+docker-compose up -d
+
+# download compiler 
+
+# search recursively through repo dir for compiler 
+find . -name "gcc-arm-11.2-2022.02-x86_64-arm-none-eabi*" > compiler_exist.txt
+
+if [ -s compiler_exist.txt ]; then
+
+	echo "Compiler files present. Skipping download"
+else
+	echo "No compiler files found. Downloading."
+	docker exec ${CONTAINER_NAME} wget https://developer.arm.com/-/media/Files/downloads/gnu/11.2-2022.02/binrel/gcc-arm-11.2-2022.02-x86_64-arm-none-eabi.tar.xz
+        
+        # TODO: update path to match latest changes 
+        #extract and move to appropriate dir 
+        echo "Extracting tar file..."
+	docker exec ${CONTAINER_NAME} tar -xf gcc-arm-11.2-2022.02-x86_64-arm-none-eabi.tar.xz -C embedded/platforms/
+
+        # rename compiler files and remove .xz 
+        docker exec ${CONTAINER_NAME} mv embedded/platforms/gcc-arm-11.2-2022.02-x86_64-arm-none-eabi ${COMPILER_DIR} && rm -f gcc-arm-11.2-2022.02-x86_64-arm-none-eabi.tar.xz 
```

Comment:
Pretty sure you can specify the destination dir when unpacking with `tar` above.
Also, same comment as before re: not executing with docker